tests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-
+find_package(bacio REQUIRED)
```

Comment:
This can remain at the top-level `CMakeLists.txt`.
You just don't link at the time of creating the nemsio library.

Reply:
Yes